.template/spec/addons/base/docker/template_spec.rb

```diff
@@ -5,6 +5,10 @@
     expect(file('Dockerfile')).to exist
   end
 
+  it 'creates the Dockerfile.mock' do
+    expect(file('Dockerfile.mock')).to exist
+  end
+
```

Comment:
Let's move this spec inside the OpenAPI tests :white_check_mark:

Reply:
I thought I moved it to OpenAPI 🤯 
I moved it in 7a73e42 ✅